config/brakeman.yml

```diff
@@ -0,0 +1,4 @@
+---
+:skip_checks:
+  - CheckPermitAttributes
+  - CheckRender
```

Comment:
Probably want review from @eanders on this

Reply:
Oh, they were moved here.  Got it.  (Assuming this is just moving inline arguments to config.

Reply:
Yes, that was the intention. The change was called out in the PR description. I usually try and capture these types of changes in the description but perhaps that's too easy to miss. Maybe it would be better if I add comments inline in the PR instead to make it easier to review

Reply:
Sorry, I missed the description, just responded to @gigxz's callout.  The description is definitely the right place.